tests/ci/codebuild/release/stage.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+env:
+  variables:
+    GO_TAG: go1.18.3
+phases:
+  # TODO: Everything in the install section should be moved into a prebuilt image
+  install:
+    runtime-versions:
+      java: corretto11
+      python: 3.9
+    commands:
+  build:
+    commands:
+      # TODO: Sign jars here
+      # TODO: Uncomment the following lines once configured
+      - pip install awscli aws-encryption-sdk-cli
+      # Debugging only
+      - aws sts get-caller-identity
+      # Retrieve encrypted configuration from S3
+      - aws s3 cp s3://${CONFIG_PREFIX}/${JKS_FILENAME}.enc $HOME/${JKS_FILENAME}.enc
```

Comment:
who sets `CONFIG_PREFIX` and the other environment variables?

Reply:
CodePipeine/CodeBuild